build-tools/create-vsix/create-vsix.csproj

```diff
@@ -7,13 +7,13 @@
     <ProjectGuid>{94756FEB-1F64-411D-A18E-81B5158F776A}</ProjectGuid>
     <OutputType>Library</OutputType>
     <RootNamespace>Xamarin.Android.Sdk</RootNamespace>
-    <AssemblyName>Xamarin.Android.Sdk-OSS</AssemblyName>
+    <AssemblyName>Xamarin.Android.Sdk</AssemblyName>
```

Comment:
Since `$(AsemblyName)` is changed, this should also change the `_GetVsixPath` target in `create-vsix.targets` to re-add `-OSS`

```xml
      <VsixPath Condition=" '$(VsixPath)' == '' ">..\..\bin\Build$(Configuration)\$(AssemblyName)-OSS-$(ProductVersion).$(XAVersionCommitCount)_$(XAVersionBranch)_$(XAVersionHash).vsix</VsixPath>
```

The reason for this is so that you can sanely download both commercial and OSS `.vsix` files into the same directory without fearing for filename collisions.